src/frontend/admin/src/components/templates/organizations/form/sections/OrganizationAddressForm.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from "react";
+import * as Yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Unstable_Grid2";
+import { FormattedMessage, useIntl } from "react-intl";
+import Typography from "@mui/material/Typography";
+import {
+  DTOOrganizationAddress,
+  Organization,
+} from "@/services/api/models/Organization";
+import { SimpleCard } from "@/components/presentational/card/SimpleCard";
+import { RHFProvider } from "@/components/presentational/hook-form/RHFProvider";
+import { RHFTextField } from "@/components/presentational/hook-form/RHFTextField";
+import { useOrganizations } from "@/hooks/useOrganizations/useOrganizations";
+import { organizationAddressFormMessages } from "@/components/templates/organizations/form/translations";
+import { RHFSelect } from "@/components/presentational/hook-form/RHFSelect";
+
+type Props = {
+  organization: Organization;
+};
+export function OrganizationAddressForm({ organization }: Props) {
+  const intl = useIntl();
+  const organizationQuery = useOrganizations({}, { enabled: false });
+  const organizationAddress = organization?.addresses?.[0];
```

Comment:
You should filter addresses to get the main one not the first one.

```suggestion
const organizationAddress = organization?.addresses?.find(address => address.is_main);
```

Reply:
Yes, but it's the same for the moment, right? We can only add one address currently